x/autopilot/types/params.go

```diff
@@ -8,12 +8,14 @@ import (
 )
 
 const (
-	// DefaultActive is the default value for the active param (set to true)
-	DefaultActive = false
+	// Default active value for each autopilot supported module
+	DefaultStakeibcActive = false
+	DefaultClaimActive    = true
```

Comment:
Wdyt of setting this to false, and overriding to true in the upgrade?

Reply:
I was wondering the same thing. I lean slightly towards setting them like this to keep our params as close to the defaults as possible (given how clunky it is to change them in the upgrade handler)

Reply:
+1 I would also keep them to the defaults

Reply:
It's just weird to have the defaults as false/true due to external circumstances